proxy/response.go

```diff
@@ -4,17 +4,19 @@ import (
 	"io"
 	"mime"
 	"net/http"
+
+	"github.com/miolini/datacounter"
 )
 
 type ResponseReader struct {
-	io.Reader
```

Comment:
We actually need this reader because transcoders (eg. gzip) might want to wrap the reader and reassign to this field for other transcoders down the chain. I guess wrapping the whole ResponseReader may be cleaner than mutating this, but this seemed more practical.